content/learn/book/storing-data/local-system-param.md

```diff
@@ -1,15 +1,51 @@
 +++
-title = "Local System Parameters"
+title = "System-Local Data Storage"
 insert_anchor_links = "right"
 [extra]
 weight = 5
 status = 'hidden'
 +++
 
-Sometimes you may need to preserve some state between executions of a [system].
-For this, we have the [`Local<T>`] system parameter.
+There's one more place where the ECS can store data: on the systems themselves.
+With the help of the [`Local<T>`] system parameter, you can cache private data from one run of a system to the next.
```

Comment:
```suggestion
With the help of the [`Local<T>`] system parameter, you can cache data from one run of a system to the next.
```
I don't like the word `private` here, as it may be conflated with visibility modifiers.